skaffold.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: skaffold/v2beta14
+kind: Config
+metadata:
+  name: metrics-server
+build:
+  artifacts:
+  - image: gcr.io/k8s-staging-metrics-server/metrics-server
+    docker:
+      dockerfile: Dockerfile
+      buildArgs:
+        GIT_TAG: "v0.4.0"
```

Comment:
since this is meant for development only, wouldn't it be better to use an unofficial tag such as `devel` or `HEAD`?

Reply:
SG